packages/clerk-js/webpack.config.js

```diff
@@ -60,6 +60,11 @@ const common = ({ mode }) => {
             name: 'zxcvbn-common',
             chunks: 'all',
           },
+          coinbaseWalletSDKVendor: {
+            test: /[\\/]node_modules[\\/](@coinbase\/wallet-sdk|ieee754|preact|keccak|buffer|string_decoder|sha\.js|base64-js|safe-buffer|util-deprecate|inherits)[\\/]/,
```

Comment:
We are manually grouping together all deps that are only used from the coinbase wallet sdk in order to avoid puting them in the vendors chunk